tls/extensions/s2n_extension_list.c

```diff
@@ -22,6 +22,10 @@
 #include "error/s2n_errno.h"
 #include "utils/s2n_safety.h"
 
+#define s2n_parsed_extension_is_valid(parsed_extension) ((parsed_extension)->extension.data != NULL)
```

Comment:
this was renamed to `s2n_parsed_extension_is_empty`, right?

Reply:
Oh right! I'll update to match